orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -45,6 +45,7 @@ struct TrainingParameters {
   int horizontal_parallel_size = 1;
   int deepspeed_zero_stage = 0;
   bool enable_grad_norm_clip = true;
+  bool enable_gelu_approximation = false;
```

Comment:
From @KeDengMS. Could you add a test case for the new option in orttraining/orttraining/test/python/orttraining_test_orttrainer_frontend.py?

Reply:
It seems it's not possible to save the optimized graph from the Python FE. I guess this is a good reason to add it? It would be useful regardless. @thiagocrepaldi Shall I make another PR on 'internal_use' for dumping graphs?

Reply:
Yes, that. I suppose the option that dumps with loss does not help much. Though the optimized training graph might be desirable. That option dumps before optimization. (The optimized one is often more readable.)